Client/src/main/java/org/example/fibonacci/ReactionGameEventLoop/EventManager.java

```diff
@@ -0,0 +1,21 @@
+package org.example.fibonacci.ReactionGameEventLoop;
```

Comment:
Package `ReactionGameEventLoop` you need to put at same level than `java/org/example`, doesn't inside of fibonacci package